pkgs/games/srb2/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchurl
+, fetchFromGitHub
+, cmake
+, curl
+, nasm
+, openmpt123
+, p7zip
+, libgme
+, libpng
+, SDL2
+, SDL2_mixer
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "srb2";
+  version = "2.28";
+  assets_version = "2.2.5";
+
+  src = fetchFromGitHub {
+    owner = "STJr";
+    repo = "SRB2";
+    rev = "SRB2_release_${version}";
```

Comment:
404 file not found.

Reply:
uhm I somehow must have changed the version number just before I commited, sorry about that...

Reply:
I am unsure why `2.28` still works for me though?